src/events/cd-event-list.vue

```diff
@@ -32,14 +32,21 @@
       </div>
       <div class="cd-event-list__event-view-wrapper">
         <router-link :to="{name: 'EventDobVerification', params: {eventId: event.id}}"
+                     v-if="!isEventFull(event.id)"
                      tag="button" class="btn btn-lg btn-primary cd-event-list__event-view">
           {{ $t('See Details and Book') }}
         </router-link>
+        <router-link :to="{name: 'EventDobVerification', params: {eventId: event.id}}"
```

Comment:
Since this is disabled, should probably just use a regular `<button>` tag instead, since the `router-link` will never be able to go anywhere anyway.